easybuild/easyblocks/t/tensorflow.py

```diff
@@ -318,13 +318,14 @@ def configure_step(self):
                 raise EasyBuildError("TensorFlow has a strict dependency on cuDNN if CUDA is enabled")
             if nccl_root:
                 nccl_version = get_software_version('NCCL')
+                nccl_maj_min_ver = '.'.join(nccl_version.split('.')[:2])
```

Comment:
This means we are reporting `2.4` as opposed to `2.4.6` (or `2.4.6-1`), are we sure this doesn't have consequences? We could be more conservative here and do
```
nccl_version = nccl_version.split('-')[0]
```

Reply:
Yeah, that's better.
I just scrapped the -x on the version when creating the NCCL ECs, didn't see a reason to keep them.

And it's not really a pre-release identifier, it's just what the OS agnostic version uses instead of "-ga"

Reply:
I did take a look  at the releases last night in https://github.com/NVIDIA/nccl/releases and there is a `2.5.6-2` in there, so I don't think they can be completely ignored

Reply:
FWIW: I checked the TF sources on what they do with this: They search the provided nccl installation paths for a nccl.h and compare the given version against the `major.minor.patch` given in the header by matching it at the start. I.e. `if "major.minor.patch".startswith(TF_NCCL_VERSION) --> OK`

Reply:
So what you would suggest we do?

Reply:
Sorry, I committed directly to the branch, I meant to make a suggestion

Reply:
Technically omitting the patch version works, but I'd stay with the more explicit approach and include it. Hence the current state with the above suggestion is fine